src/Asset/EventSubscriber/DeletionSubscriber.php

```diff
@@ -49,18 +49,21 @@ public function onStateChanged(JobRunStateChangedEvent $event): void
         if ($event->getJobName() !==  Jobs::DELETE_ASSETS->value) {
             return;
         }
+        $ownerId = $event->getJobRunOwnerId();
```

Comment:
I don't think an extra variable is necessary right? We have it all encapsulated in the event already.

WDYT?

Reply:
sounds good :)